src/components/MahjongSelector.tsx

```diff
@@ -76,6 +84,8 @@ const MahjongSelector: React.FC = () => {
     // Check if the tile is already in the map
     const count = newSelectedTile.get(tile) || 0;
 
+    if (count === 0) return;
```

Comment:
I'm not sure about this code.
Because the purpose is just to check whether the card deselection operation is valid. If this line of code is redundant, I can omit it.